roles/testnode/tasks/zypper_systems.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Tasks common to all systems that use the zypper
+# package manager
```

Comment:
Should we add a comment here along the lines of "this is mostly a copy of yum_systems.yml", just to make any decisions about later changes slightly easier?